daemons/based/based_messages.c

```diff
@@ -353,17 +353,6 @@ cib_process_replace_svr(const char *op, int options, const char *section, xmlNod
     return rc;
 }
 
-/* @COMPAT: Remove when PCMK__CIB_REQUEST_ABS_DELETE is removed
```

Comment:
Ouch, I totally zoned out on that one. Dropping the commit

Reply:
I don't think it's a big deal in practice: it would have always returned `-EINVAL` and with this commit it would return `-EOPNOTSUPP` and do less logging. I'm fine either way as long as we're aware.

Reply:
If we keep this request type, we should update the comment. It's misleading to point only to remote nodes. See comment about rolling upgrade

Reply:
will do in a future pr